Packs/AWS-S3/Integrations/AWS-S3/AWS-S3.py

```diff
@@ -191,14 +191,26 @@ def list_objects_command(args, aws_client):
         role_session_duration=args.get('roleSessionDuration'),
     )
     data = []
-    response = client.list_objects(Bucket=args.get('bucket'))
-    if response.get('Contents', None):
-        for key in response['Contents']:
-            data.append({
-                'Key': key['Key'],
-                'Size': convert_size(key['Size']),
-                'LastModified': datetime.strftime(key['LastModified'], '%Y-%m-%dT%H:%M:%S')
-            })
+    kwargs = {
+        'Bucket': args.get('bucket')
+    }
+    if args.get('delimiter') is not None:
+        kwargs.update({'Delimiter': args.get('delimiter')})
+    if args.get('prefix') is not None:
+        kwargs.update({'Prefix': args.get('prefix')})
+
+    client.list_objects(**kwargs)
+    paginator = client.get_paginator('list_objects')
+    for response in paginator.paginate(**kwargs):
```

Comment:
Needed to convert to kwargs to support the pagination here. Same implementation as we use in EC2